commitlint.config.js

```diff
@@ -0,0 +1,21 @@
+const { RuleConfigSeverity } = require('@commitlint/types');
+const {
+  utils: { getProjects },
+} = require('@commitlint/config-nx-scopes');
+
+/** @type {import('@commitlint/types').UserConfig} */
+const configuration = {
+  extends: ['@commitlint/config-conventional'],
+  rules: {
+    'scope-enum': async ctx => {
```

Comment:
Should we consider a scope for root level and general things like GH actions etc?

Reply:
Sounds good!

scopes:
- `tools`
- `workflows`
- `testing` => for generic testing helper?

Reply:
Done :heavy_check_mark: